src/main/java/asynchronous/typing/Leaderboard.java

```diff
@@ -71,7 +71,8 @@ public void run()
 			if (position - 20 >= initialPosition) {break;}
 			
 			try {
-				userTag = jda.retrieveUserById( user.getString("_id") ).complete().getAsTag();
+				//TODO: preferably get rid of jda call altogether as it is astonishingly slow
```

Comment:
I wonder if there is any real way to do this, considering that we need to retrieve users somehow, and as far as I know, this is the only way of doing it without omitting much of the leaderboard in any one server.

For the time being, since we are only going to be calling this very occasionally (most likely during startup, and a few other instances where a new user submits a test), I think we might be fine.